### PR TITLE
feature/Create controllers

### PR DESCRIPTION
**Pull request by NitBravoA92**
# 🚩Milestone 4: Controllers | create-controllers - >>branch<< -

### Hi there! @NitBravoA92 here. 👋

This is the PR for the **Blog app - controllers** activity. The main goal of this milestone is to use the controllers to handle requests and render empty views.

### Here is a summary of what has been done:

- [x] Create the routes for `users` using `resources`

- [x] Create the `users` controller with two action methods: `index` and `show`.

- [x] Create the `index` and `show` views for the users' controller

- [x] Create the routes for `posts` using `resources`

- [x] Create the `posts` controller with two action methods: `index` and `show`.

- [x] Create the `index` and `show` views for the posts' controller

- [x] Update the README.md file

### Results of the implementation of the routes:
Captura desde 2023-09-25 21-00-49

## ⭐To the code reviewer 👨‍💻

💝 I really thank 🙇‍♂️ you, dear code reviewer 👨‍🎨 for dedicating your precious 🥇 time ⌚ to check the PR 🧾 .

I kindly ask 🙏 dear code reviewer 🤵 that if there is any issue 🦯 in this project, please do list 📃 them in a descriptive 💡 manner and give your best suggestions 🎁 if needed.

If you think some significant issues need to be changed ♻️ please kindly contact 📞 me through Zoom or Slack, I am available on Microverse UTC-6 time zone.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # Adding nested resources routes for users and posts
+  resources :users do
+    resources :posts
+  end
```
Comment (Abenezer-Tilahun) [inline config/routes.rb]: - [x] As stated in the project requirement kindly make sure that you added code to handle only the **4 required URLs.** when I run `rails routes` command in my console, I got the the following output
route

- But it should display only the **4 required URLs.** just like the below result 
route correct

- So kindly modify the` routes.rb` just like the below sample to get the required result 

```
Rails.application.routes.draw do
  # Adding nested resources routes for users and posts
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
 

  # Defines the root path route ("/")
  # root "articles#index"
end
```

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @NitBravoA92 

Great job for implemented  the some of the Project Requirement  , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git-Flow
✔️ Good commit messages
✔️ Professional README
✔️ Correctly create: A route, An action in the correct controller and A view file for each URL.


almost

## Required Changes ♻️
_Check the comments below the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (NitBravoA92) [reply config/routes.rb]: Hello @Abenezer-Tilahun Thank you very much for your suggestions! I have applied the changes.

**Review by Reem-lab — APPROVED** [review]
# Hi @NitBravoA92,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  Followed Gitflow 💯
- [ ]  Descriptive PR⭐
- [ ]  Kudos for working on the controllers🤩
- [ ]  No Linters error ✅



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**